scipy/ndimage/tests/test_interpolation.py

```diff
@@ -538,418 +654,459 @@ def test_map_coordinates_endianness_with_output_parameter(self):
             result = out if returned is None else returned
             assert_array_almost_equal(result, expected)
 
-    def test_map_coordinates_with_string_output(self):
-        data = np.array([[1]])
+    def test_map_coordinates_with_string_output(self, xp):
+        if not is_numpy(xp):
+            pytest.xfail("string output is numpy-only")
```

Comment:
again, please use `skip_xp_backends(np_only=True, reasons=['string output is NumPy-only'])`